env.example

```diff
@@ -8,3 +8,6 @@ REACT_APP_GITHUB_TOKEN = "YOUR GITHUB TOKEN HERE"
 GITHUB_USERNAME = "YOUR GITHUB USERNAME HERE"
 // Set to true to fetch profile data from GitHub (remember to remove all components relying on GitHub data if set to false)
 USE_GITHUB_DATA = "true"
+// Set to true to fetch blog data from Medium (otherwise, hardcoded values from Blogs.js are used)
+USE_MEDIUM_DATA = "true"
+MEDIUM_USERNAME = "YOU MEDIUM USERNAME HERE"
```

Comment:
Can you update the corresponding docs in README.md?

Reply:
done: https://github.com/saadpasta/developerFolio/pull/445/commits/176cfb186e8c74ec68002c11ee651f0894e6174b